RELEASENOTES.md

```diff
@@ -0,0 +1,3 @@
+### New in 5.0.0
```

Comment:
So, an alternative idea then, perhaps it would be useful to keep an empty release notes file and enforce it has contents on release? Then as the release finishes, the file is emptied again?